code/game/machinery/autolathe/nanoforge.dm

```diff
@@ -0,0 +1,100 @@
+/obj/machinery/autolathe/nanoforge
+	name = "matter nanoforge"
+	desc = "It consumes items and produces compressed matter."
+	icon_state = "nanoforge"
+	icon = 'icons/obj/machines/autolathe.dmi'
+	use_oddities = TRUE
+	is_nanoforge = TRUE
+	circuit = /obj/item/weapon/electronics/circuitboard/nanoforge
+	var/list/nanoforge_designs = list()
+	var/list/tags_to_spawn = list(SPAWN_DESING)
```

Comment:
With https://github.com/discordia-space/CEV-Eris/pull/5832 merged, change to SPAWN_DESIGN

Reply:
that's not true.
#5797